### DISCUSSION TITLE
Greaseweazle unknown mark f9 & ibm empty from raw flux

### DISCUSSION THREAD

**Discussion by Budghiss** [keirf/greaseweazle#458] [General]
Hi, and thanks for the time to help. New user (2 weeks) and everything was going smoothly until I received a bad disk that I didn't eject right away. The original floppy drive that read the bad disk was a Sony 920 - it started to error during the read session  'IBM Empty from Raw Flux' throughout every sector. I reset the gw, and same result. I rebooted, same result. I then changed the floppy drive to a good TEAC 235. Now when go to read, I get sporadic reads, but a long series of 'Unknown Mark F9'. I can't find much documentation on this, and I am not a programmer. Does gw log to a file to look at, or could someone please point me in the right direction?

**Comment by Budghiss** [comment]
BTW, I am on a Linux Mint box, and the gw appears to erase properly, but not write...errors out verifying tracks...

**Comment by keirf** [comment]
So can you read good disks still, using a good drive? The problem with dirty disks is that they make your good drives dirty (and can even break them).

**Reply by Budghiss** [reply]
No, my second drive is producing errors of 'Unknown Mark xx' sporadically through the read, making it useless...

**Reply by Budghiss** [reply]
Also, when I attempt to write verified images (scp & img), it errors out verifying the tracks...

**Reply by keirf** [reply]
If you can't read, you won't be able to verify either. It sounds like you don't have a good drive. Maybe do a raw dump with the 235 drive and share. Eg gw read name.scp

**Reply by Budghiss** [reply]
Maybe I'm overreacting - is the 'Unknown Mark f9' error part of logging - does it affect the image? I put a third drive in, another Sony 920, and it read a disk I knew was good 100%, and I don't believe there were 'Unknown Mark f9' errors...

**Reply by Budghiss** [reply]
This 3rd drive is working properly it seems - no more 'Unknown Mark ' errors - Looks like it got my drive - I guess just let it go...Thanks for the input...

**Reply by keirf** [reply]
Yeah, the unknown mark messages are just warnings and usually indicate marginal disk read that is close to correct but with frequent bit errors. Means clean the disk or drive or try another.
